src/main/java/com/contrastsecurity/maven/plugin/sdkx/DefaultContrastScanSDK.java

```diff
@@ -38,6 +40,46 @@ public DefaultContrastScanSDK(final ContrastSDK contrast, final String restURL)
     this.restURL = ContrastSDKUtils.ensureApi(Objects.requireNonNull(restURL));
   }
 
+  @Override
+  public Project createProject(final String organizationId, final CreateProjectRequest request)
+      throws IOException, UnauthorizedException {
+    // requests made with ContrastSDK.makeRequest must have their path prepended with "/"
+    final String path = String.join("/", "", "sast", "organizations", organizationId, "projects");
+    final String json = gson.toJson(request);
+    try (Reader reader =
+        new InputStreamReader(
+            contrast.makeRequestWithBody(HttpMethod.POST, path, json, MediaType.JSON))) {
+      return gson.fromJson(reader, Project.class);
+    }
+  }
+
+  @Override
+  public Project findProjectByName(final String organizationId, final String projectName)
+      throws IOException, UnauthorizedException {
+    // requests made with ContrastSDK.makeRequest must have their path prepended with "/"
+    final String uri =
+        String.join("/", "", "sast", "organizations", organizationId, "projects")
+            + "?unique=true&name="
```

Comment:
`unique=true` implies that you can have multiple projects with the same name. Seems like a lot of potential for confusion there. Maybe that’s another area where the umbrella API can improve? Or maybe I’m misunderstanding.

Reply:
Yes, that was an issue, and the team resolved it. This `unique` parameter is another change that team made for us. the name is imperfect, but it means "perform an exact match on the project name instead a 'contains' match". Without this parameter, if you search for "t", you won't get a project named "t", you'll get every project that starts with "t". The `unique` parameter is the only way they give us to look-up a project by name.

I'll send you more context in Slack